vnext/src/Libraries/Components/Picker/PickerWindows.tsx

```diff
@@ -19,7 +19,7 @@ const RCTPicker = requireNativeComponent<
 
 class PickerItem extends React.Component<IPickerItemProps> {
   public render(): JSX.Element | null {
-    return null;
+    throw null;
```

Comment:
`throw null` is kind of crazy, but matches upstream behavior/behavior in Picker.js